### PR TITLE
CATROID-15 implement color touches color

### PR DESCRIPTION
**Pull request by gPathpp**
refactor touches color
implement Color touches Color

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: catroid/src/main/java/org/catrobat/catroid/common/Constants.java [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
 public final class Constants {
 
-	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = 0.9999994f;
+	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = 1f;
```
Comment (adnaribo) [inline catroid/src/main/java/org/catrobat/catroid/common/Constants.java:42]: @wslany please decide if we can move to 1

**Review by adnaribo — APPROVED** [review]
looks good to me :)

**Review by wslany — CHANGES_REQUESTED** [review]
Thanks!!

As discussed via slack (for others: there were cases where it didn't report the correct value --- was easier to show than explain here).

**Review by wslany — APPROVED** [review]

**Comment by wslany** [comment]
Thanks!!!

**Pull request merged**